### PR TITLE
enable sq4&sq4_uniform in HGraph

### PR DESCRIPTION
**Pull request by LHT129**

### CODE REVIEW

**Review by inabao — APPROVED** [review]
LGTM

File: src/index/hgraph_zparameters.cpp [hunk 1]
```diff
@@ -60,12 +60,17 @@ HGraphParameters::CheckAndSetKeyValue(const std::string& key, JsonType& value) {
 
     if (key == HGRAPH_BASE_QUANTIZATION_TYPE) {
         std::string value_str = value;
-        if (value_str != QUANTIZATION_TYPE_VALUE_SQ8 && value_str != QUANTIZATION_TYPE_VALUE_FP32) {
-            throw std::invalid_argument(fmt::format("parameters[{}] must be {} or {}, now is {}",
-                                                    HGRAPH_BASE_QUANTIZATION_TYPE,
-                                                    QUANTIZATION_TYPE_VALUE_SQ8,
-                                                    QUANTIZATION_TYPE_VALUE_FP32,
-                                                    value_str));
+        if (value_str != QUANTIZATION_TYPE_VALUE_SQ8 && value_str != QUANTIZATION_TYPE_VALUE_FP32 &&
```
Comment (jiaweizone) [inline src/index/hgraph_zparameters.cpp:63]: indent

Comment (LHT129) [reply src/index/hgraph_zparameters.cpp:63]: clang-format

**Review by jiaweizone — APPROVED** [review]
LGTM

**Pull request merged**